disnake/errors.py

```diff
@@ -116,16 +116,17 @@ def __init__(
         if isinstance(message, dict):
             self.code = message.get("code", 0)
             base = message.get("message", "")
-            errors = message.get("errors")
-            if errors:
-                errors = _flatten_error_dict(errors)
-                helpful = "\n".join(f"In {k}: {m}" for k, m in errors.items())
+            self.errors = message.get("errors")
+            if self.errors:
+                flattened_errors = _flatten_error_dict(self.errors)
```

Comment:
```suggestion
            self._errors = message.get("errors")
            if self._errors:
                flattened_errors = _flatten_error_dict(self._errors)
```
This should be an implementation detail.
(also needs to be changed in a couple other locations)